cloud/endagaweb/models.py

```diff
@@ -103,15 +104,20 @@ def new_user_hook(sender, instance, created, **kwargs):
         """
         if created and instance.username != settings.ANONYMOUS_USER_NAME:
             profile = UserProfile.objects.create(user=instance)
-            network = Network.objects.create()
+
+            # To avoid duplicate names while running setup_test_db
+            # network = Network.objects.create()
+            network = Network.objects.create(name='Network_%s' % instance.pk)
```

Comment:
When passing a single value to string formatting operator it's better to pass it as a 1-tuple: `(instance.pk, )`